package programs;

public class ExceptionExample1 {

	public static void main(String[] args)
	{
		int x=20,y=10,z;
		try
		{
		z= x/(y-y);
		System.out.println("Division is:"+z);
		}
		catch(Exception obj)
		{
			
		}
		
		
		z=x*y;
		System.out.println("\n\nMultiplication is:"+z);
		
		
		

	}

}
